src/types/user-course.type.ts

```diff
@@ -0,0 +1,10 @@
+export type UserCourse = {
+  id?: number
+  userId: number
+  courseId: number
+  dropped?: boolean
+  level?: string
```

Comment:
If this can be only certain strings then this type should only be 

`'instructor' | 'ta' | 'student'`

Reply:
You can do that?? TpyeScript ftw!